frontend/public/components/utils/dropdown.jsx

```diff
@@ -715,6 +715,7 @@ ActionsMenu.propTypes = {
       accessReview: PropTypes.object,
     }),
   ).isRequired,
+  impersonate: PropTypes.any,
```

Comment:
@fhlavac was this prop type definition missing prior to your change? Looks fine, just wanted to confirm this is intentional.

Reply:
@florkbr I've added it because of some warning during the build but in current state, it doesn't seem to be necessary. Removed, thank you